web/src/components/wizards/S3LogSourceWizard/ValidationPanel/SuccessContent.tsx

```diff
@@ -0,0 +1,147 @@
+/**
+ * Panther is a Cloud-Native SIEM for the Modern Security Team.
+ * Copyright (C) 2020 Panther Labs Inc
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+import { useWizardContext, WizardPanel } from 'Components/Wizard';
+import { AbstractButton, Button, Flex, Img, Link, SimpleGrid, Spinner } from 'pouncejs';
+import SuccessStatus from 'Assets/statuses/success.svg';
+import LinkButton from 'Components/buttons/LinkButton';
+import urls from 'Source/urls';
+import React from 'react';
+import { Link as RRLink } from 'react-router-dom';
+import { useFormikContext } from 'formik';
+import { S3LogSourceWizardValues } from 'Components/wizards/S3LogSourceWizard/S3LogSourceWizard';
+import { useGetS3LogSourceLazyQuery } from 'Pages/EditS3LogSource/graphql/getS3LogSource.generated';
+import HealthCheckWarning from './HealthCheckWarning';
+
+interface SuccessContentProps {
+  integrationId: string;
+}
+
+const SuccessContent: React.FC<SuccessContentProps> = ({ integrationId }) => {
+  const { reset: resetWizard } = useWizardContext();
+  const { initialValues, resetForm } = useFormikContext<S3LogSourceWizardValues>();
+
+  const [getS3, { data, loading }] = useGetS3LogSourceLazyQuery({
```

Comment:
can be a simple query. Just use `refetch`
```suggestion
  const { data, loading, refetch } = useGetS3LogSource({
```

Reply:
Good call! Done.